nrf-hal-common/src/uarte.rs

```diff
@@ -368,9 +386,18 @@ pub enum Error {
     BufferNotInRAM,
 }
 
-pub trait Instance: Deref<Target = uarte0::RegisterBlock> {}
+pub trait Instance: Deref<Target = uarte0::RegisterBlock> {
+    #[cfg(feature = "bbq-uarte")]
```

Comment:
I put this behind a flag because it's technically a breaking change I guess? I could be convinced it isn't though